tests/integration/test_sqs.py

```diff
@@ -201,6 +201,14 @@ def test_send_receive_message(self, sqs_client, sqs_queue):
         assert message["MessageId"] == send_result["MessageId"]
         assert message["MD5OfBody"] == send_result["MD5OfMessageBody"]
 
+    @pytest.mark.aws_validated
+    def test_send_oversized_message(self, sqs_client, sqs_queue):
+        try:
+            sqs_client.send_message(QueueUrl=sqs_queue, Message=(262145 * "a"))
+        except ClientError as e:
+            assert "InvalidMessageContents" in e.response["Error"]["Code"]
+            assert e.response["ResponseMetadata"]["HTTPStatusCode"] in [400, 404]
```

Comment:
i can see we are using this pattern in other tests, but we are becoming more strict also with error codes, so it would be great to just match exactly what AWS returns, which for InvalidMessageContents is 400.

it would also be great if we could match the exact message as it returns. bonus points for writing a [snapshot test](https://localstack.cloud/blog/2022-08-04-parity-explained/#parity-tests-with-snapshot-testing) instead of using plain asserts ;-)  but optional

Reply:
I will make these changes. Thank you for the excellent feedback.